src/qibo/models/error_mitigation.py

```diff
@@ -0,0 +1,35 @@
+'''
```

Comment:
@shangtai I believe this file can be removed, right?

Reply:
yes, it can be removed.